pkg/sql/sqlstats/cluster_settings.go

```diff
@@ -105,7 +105,7 @@ var MaxMemReportedSQLStatsStmtFingerprints = settings.RegisterIntSetting(
 	settings.ApplicationLevel,
 	"sql.metrics.max_mem_reported_stmt_fingerprints",
 	"the maximum number of reported statement fingerprints stored in memory",
-	100000,
+	10000,
```

Comment:
how did this sneak in?

Reply:
Can you just remove the benchmark commit from the backport since it's superfluous?

Reply:
I think you're looking at that one commit. I probably just amended the worng commit when I was reverting this change.